cordformation/src/test/kotlin/net/corda/plugins/CordformTest.kt

```diff
@@ -17,7 +17,7 @@ class CordformTest {
     private lateinit var buildFile: File
 
     private companion object {
-        const val cordaFinanceJarName = "corda-finance-3.0-SNAPSHOT"
+        const val cordaFinanceJarName = "corda-finance-3.1-corda"
```

Comment:
This isn't a great place for the version number - isn't this dependent on `$project.version`?